src/components/address-book/ImportDialog/index.tsx

```diff
@@ -86,12 +84,18 @@ const ImportDialog = ({ handleClose }: { handleClose: () => void }): ReactElemen
             setZoneHover(false)
             setError(undefined)
 
-            const message = abOnUploadValidator(result)
+            // Remove empty rows
+            const cleanResult = {
+              ...result,
+              data: result.data.filter(hasEntry),
+            }
```

Comment:
Added this because some editors automatically add an empty new line at the end of file. We should just ignore them.